evm-subql/package.json

```diff
@@ -9,8 +9,8 @@
     "prepack": "rm -rf dist && npm build",
     "codegen": "./node_modules/.bin/subql codegen",
     "test": "echo",
-    "index": "subql-node -f . --local --batch-size 200 --subquery-name=acala-evm --debug",
-    "query": "PORT=3001 subql-query --name acala-evm --playground --indexer=http://localhost:3000 --debug"
+    "index": "yarn subql-node -f . --local --batch-size 200 --subquery-name=acala-evm  --port 3030",
```

Comment:
by the way, did it run for you locally? the official doc recommended installing them locally, and I remember using local subql lib had some issue before. If this works locally now then it's ok, just wanna make sure it can run

Reply:
Yes, it can run.

Reply:
export DB_USER=postgres
export DB_PASS=postgres
export DB_DATABASE=postgres
export DB_HOST=localhost
export DB_PORT=5432

yarn query